### PR TITLE
Add remove hard bounce flag endpoint

### PR DESCRIPTION
**Pull request by brrusselburg**
Also updated docs for future me (lol) so that VCR use is a bit easier to navigate in this repo.

This is for https://reverb.atlassian.net/jira/software/projects/MTECH/boards/21?selectedIssue=MTECH-887

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,6 +266,14 @@ export BRAZE_RUBY_DEBUG=true
 bundle exec rails whatever
 ```
 
+### VCR
```
Comment (loudmouse) [inline README.md:269]: Thanks for adding these. Future me will appreciate these. 👍

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.6.7
```
Comment (meshulam) [inline .ruby-version]: let's keep this out of the codebase, or at least separate it into its own commit. If we open a PR to upstream they may not want this if they aren't using rbenv.

**Review by Jhowden — APPROVED** [review]

File: .ruby-version (hunk 2 above)
Comment (brrusselburg) [reply .ruby-version]: sounds good, I'll just revert it back. was easy enough to add.
